### PR TITLE
Add pagerank testcase

### PR DESCRIPTION
**Pull request by gczsjdy**

### CODE REVIEW

File: bin/workloads/websearch/pagerank/prepare/prepare.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+enter_bench HadoopPreparePagerank ${workload_config}
+show_bannar start
+
+rmr-hdfs $INPUT_HDFS || true
+START_TIME=`timestamp`
+
+OPTION="-t pagerank \
+        -b ${PAGERANK_BASE_HDFS} \
+        -n ${INPUT_HDFS} \
```
Comment (carsonwang) [inline bin/workloads/websearch/pagerank/prepare/prepare.sh]: This doesn't seem to be correct. Did you check the generated data?

File: pegasus/pom.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    <version>2.0-SNAPSHOT</version>
+    <name>PEGASUS: A Peta-Scale Graph Mining System</name>
+    <url>http://www.cs.cmu.edu/~pegasus/index.htm</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-core</artifactId>
+            <version>${hadoop.mr1.version}</version>
```
Comment (carsonwang) [inline pegasus/pom.xml]: Can we use hadoop.mr2.version?

File: sparkbench/websearch/pom.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    <artifactId>sparkbench</artifactId>
+    <version>6.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>com.intel.hibench.sparkbench</groupId>
+  <artifactId>sparkbench-websearch</artifactId>
+  <packaging>jar</packaging>
+  <version>6.0-SNAPSHOT</version>
+  <name>sparkbench websearch benchmark</name>
+-->
```
Comment (carsonwang) [inline sparkbench/websearch/pom.xml]: Let's remove these.

File: sparkbench/websearch/pom.xml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-core_${scala.binary.version}</artifactId>
+      <version>${spark.version}</version>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+  <build>
```
Comment (carsonwang) [inline sparkbench/websearch/pom.xml]: The parent pom already includes `build` part. This can be removed?

**Comment by carsonwang** [comment]
Can you also resolve the conflicts?

File: sparkbench/websearch/pom.xml (hunk 4 above)
Comment (gczsjdy) [inline sparkbench/websearch/pom.xml]: All done ~

File: bin/workloads/websearch/pagerank/prepare/prepare.sh (hunk 1 above)
Comment (gczsjdy) [inline bin/workloads/websearch/pagerank/prepare/prepare.sh]: Thanks for all the review work you've done, I will be more careful next time. ^_^

**Pull request merged**